applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClientBuilder.java

```diff
@@ -40,16 +40,16 @@
     private static final String ACCEPT_HEADER = "Accept";
     private static final String ACCEPT_HEADER_VALUE = "application/vnd.microsoft.azconfig.kv+json";
 
-    private final List<HttpPipelinePolicy> policies;
-    private final HttpHeaders headers;
-
-    private ConfigurationClientCredentials credentials;
-    private URL serviceEndpoint;
-    private HttpClient httpClient;
-    private HttpLogDetailLevel httpLogDetailLevel;
-    private HttpPipeline pipeline;
-    private RetryPolicy retryPolicy;
-    private String userAgent;
+    final List<HttpPipelinePolicy> policies;
```

Comment:
Do these need to be package-private? I don't see use of them outside of this class.

Reply:
Leftovers from the subclass prototype, changed to private.